src/pages/programs/multiply-two-numbers.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Java Program to Multiply Two Integers
+shortTitle: Multiply Two Integers
+description: In this program, you'll learn to store and multiply two integer numbers in Java. After multiplication, the final value is displayed on the screen.
+---
+
+import { List, ListItemGood } from '@/components/List'
+
+
+To understand this example, you should have the knowledge of the following Java programming topics:
+
+- [Java Operators](/docs/operators)
+- [Java Basic Input and Output](/docs/basic-input-output)
+
+## Multiply Two Integers
+
+A Java program that multiply two numbers predefined by the user is as follows:
+
+### Example 1: Program to Multiply Two Integers
+
+```java
+class Main {
+
+  public static void main(String[] args) {
+
+    System.out.println("Enter two numbers");
+    int first = 10;
+    int second = 2;
+
+    System.out.println(first + " " + second);
+
+    // multiply two numbers
+    int multiplication = first * second;
+    System.out.println("Multiplication: " + first + " * " + second + " = " + multiplication);
+  }
+}
+```
+
+#### Output
+
+```text
+Enter two numbers
+10 2
+Multiplication: 10 * 2 = 20
+```
```

Comment:
If you know the Scanner class then it would be better to rewrite the code and output as per the Scanner class